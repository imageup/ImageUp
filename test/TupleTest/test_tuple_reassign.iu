function main() -> int
{
	tuple : tu;
	int : i = 0;
	for(i = 0; i < 3; i=i+1)
	{
		tu = (i, i+1, i+2);
		prints("tuple value: ");
		printf(tu[0]);
		printf(tu[1]);
		printf(tu[2]);
	}
	prints("correct output: (0, 1, 2), (1, 2, 3), (2, 3, 4)");
	return 0;
}