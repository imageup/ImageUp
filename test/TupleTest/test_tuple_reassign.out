tuple value: 
0
1
2
tuple value: 
1
2
3
tuple value: 
2
3
4
correct output: (0, 1, 2), (1, 2, 3), (2, 3, 4)
